### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'data'

### ISSUE THREAD

**Issue by pigletfly** [mnubo/kubernetes-py#3]
in K8sObject
```
    def list(self):
        state = HttpRequest(method='GET', host=self.config.get_api_host(), url=self.base_url).send()
        if not state.get('status'):
            raise Exception('Could not fetch list of objects of type: {this_type}.'.format(this_type=self.obj_type))
        return state.data
```
I think it should be 
```
    def list(self):
        state = HttpRequest(method='GET', host=self.config.get_api_host(), url=self.base_url).send()
        if not state.get('status'):
            raise Exception('Could not fetch list of objects of type: {this_type}.'.format(this_type=self.obj_type))
        return state.get('data')
```
the state is a dict.

**Comment by sebastienc** [comment]
You're right, I'll fix it today.

**Comment by sebastienc** [comment]
It now returns the list of items from the Kubernetes API server. Eventually, every object type should return a list of python objects for this list but that will be in a future revision :) Commit is available in version 0.21

**Comment by pigletfly** [comment]
test
